tools/Data_explo/11_gala_autocorr.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="tool_autocorr" name="Auto-correlation" version="0.0">
+    <description>Show if there is auto-correlation</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="Explo_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/Exe11_autocorr.r'
+            '$input'
+            '$variable'
+            '$separator'
+            '$decimal'
+            '$colnames'
+            '$output'
+        ]]>
+        </command>
+        <inputs>
+            <param name="input" type="data" format="txt" label="Input table"/>

```

Comment:
the format should be csv I think. But could we change all tools to accept tsv files? Galaxy has so many tools that can work on tsv natively and we would save a conversion step.